src/components/lyne-text-input/lyne-text-input.stories.js

```diff
@@ -0,0 +1,462 @@
+  'input-type': 'email',
+  'label': 'Email address'
+};
+
+TextInputEmail.documentation = {
+  title: 'Text Input Email Address'
+};
+
+export const TextInputExistingPassword = Template.bind({});
+
+TextInputExistingPassword.argTypes = defaultArgTypes;
+TextInputExistingPassword.args = {
+  ...defaultArgs,
+  'input-autocomplete-value': inputAutoCompleteValue.options[24],
+  'input-name': 'current-password',
+  'input-placeholder': '',
+  'input-required': true,
+  'input-type': 'password',
+  'label': 'Current Password'
+};
+
+TextInputEmail.documentation = {
+  title: 'Text Input Existing Password'
+};
+
+export const TextInputNewPassword = Template.bind({});
+
+TextInputNewPassword.argTypes = defaultArgTypes;
+TextInputNewPassword.args = {
+  ...defaultArgs,
+  'input-autocomplete-value': inputAutoCompleteValue.options[52],
+  'input-name': 'new-password',
+  'input-placeholder': '',
+  'input-required': true,
+  'input-type': 'password',
+  'label': 'New Password'
+};
+
+TextInputEmail.documentation = {
+  title: 'Text Input New Password'
+};
+
+export const TextInputPhoneNumber = Template.bind({});
+
+TextInputPhoneNumber.argTypes = defaultArgTypes;
+TextInputPhoneNumber.args = {
+  ...defaultArgs,
+  'input-autocomplete-value': inputAutoCompleteValue.options[44],
+  'input-name': 'phone',
+  'input-placeholder': '',
+  'input-required': true,
+  'input-type': 'tel',
+  'label': 'Phone Number'
+};
+
+TextInputPhoneNumber.documentation = {
+  title: 'Text Input Phone Number'
+};
+
+export const TextInputOneTimeCode = Template.bind({});
+
+TextInputOneTimeCode.argTypes = defaultArgTypes;
+TextInputOneTimeCode.args = {
+  ...defaultArgs,
+  'input-autocomplete-value': inputAutoCompleteValue.options[37],
+  'input-name': 'otp',
+  'input-placeholder': '',
+  'input-required': true,
+  'input-type': 'text',
+  'label': 'SMS-Code'
+};
+
+TextInputOneTimeCode.documentation = {
+  title: 'Text Input One Time Code'
+};
+
+export const TextInputSearch = Template.bind({});
+
+TextInputSearch.argTypes = defaultArgTypes;
+TextInputSearch.args = {
+  ...defaultArgs,
+  'input-autocomplete-value': inputAutoCompleteValue.options[36],
+  'input-name': 'search',
+  'input-placeholder': '',
+  'input-required': true,
+  'input-type': 'search',
+  'label': 'Search'
+};
+
+TextInputSearch.documentation = {
+  title: 'Text Input Search'
+};
+
+export const TextInputUrl = Template.bind({});
+
+TextInputUrl.argTypes = defaultArgTypes;
+TextInputUrl.args = {
+  ...defaultArgs,
+  'input-autocomplete-value': inputAutoCompleteValue.options[53],
+  'input-name': 'url',
+  'input-placeholder': '',
+  'input-required': true,
+  'input-type': 'url',
+  'label': 'Website'
+};
+
+TextInputUrl.documentation = {
+  title: 'Text Input Url'
+};
+
+export default {
+  decorators: [
+    (Story) => (
+      <div style={'padding: 2rem'}>
+        <Story/>
+      </div>
+    )
+  ],
+  documentation: {
+    disableArgs: ['someArgToDisableForDocumentationPlatform']
```

Comment:
is just an example from the boilerplate. you can remove it since you don't need to ommit args here for the doc's page